README.rst

```diff
@@ -71,6 +71,17 @@ Binaries for Python Extension Packages <http://www.lfd.uci.edu/~gohlke/pythonlib
 
 .. _Anaconda: https://www.continuum.io/downloads
 
+Error Reporting
+--------------
+
+In case of API usage errors SoundFile raises the usuall `ValueError` or `TypeError`.
```

Comment:
I would write

    the ``soundfile`` module

instead of

    SoundFile

... but that's of course a matter of taste.